### PR TITLE
Updating commands documentation and help message

### PR DESCRIPTION
**Pull request by lucasncabral**

### CODE REVIEW

File: docs/mlgit_commands.md [hunk 1]
```diff
@@ -1159,9 +1161,12 @@ ml-git repository storage del minio
 <br>
 
 ```
-Usage: ml-git repository update
+Usage: ml-git repository update [OPTIONS]
 
-  This command updates the metadata for all entities.
+  This command will update all ml-entities metadata repository.
```
Comment (Vieira-Carlos) [inline docs/mlgit_commands.md:1166]:
```suggestion
  This command will update all ml-entities' metadata repository.
```
or maybe?

>This command will update the metadata in the local repository for all ml-entities

File: ml_git/commands/remote.py [hunk 2]
```diff
@@ -71,7 +71,7 @@ def repo_remote_md_add(**kwargs):
     repositories[MODELS].repo_remote_add(MODELS, kwargs['remote_url'], kwargs['global'])
 
 
-@repo_remote_ds.command('del', help='Remove remote dataset metadata REMOTE_URL from this ml-git repository')
+@repo_remote_ds.command('del', help='Remove remote datasets metadata REMOTE_URL from this ml-git repository')
```
Comment (Vieira-Carlos) [inline ml_git/commands/remote.py:74]:
```suggestion
@repo_remote_ds.command('del', help='Remove the REMOTE_URL datasets' metadata from this ml-git repository')
```

File: docs/mlgit_commands.md (hunk 1 above)
Comment (lucasncabral) [reply docs/mlgit_commands.md]: Suggestion applied at [5b89f59](https://github.com/HPInc/ml-git/pull/255/commits/5b89f5963b820f995fdbc23e81644eebf52ef6db).

File: ml_git/commands/remote.py (hunk 2 above)
Comment (lucasncabral) [reply ml_git/commands/remote.py]: Suggestion applied at [5b89f59](https://github.com/HPInc/ml-git/pull/255/commits/5b89f5963b820f995fdbc23e81644eebf52ef6db).

**Review by Vieira-Carlos — APPROVED** [review]

**Review by andersonpablosilva — APPROVED** [review]

**Pull request merged**